src/EFCore.Relational/Query/SqlExpressions/ColumnExpression.cs

```diff
@@ -10,6 +10,15 @@
 
 namespace Microsoft.EntityFrameworkCore.Query.SqlExpressions
 {
+    /// <summary>
+    ///     <para>
+    ///         An expression that represents SQL column.
```

Comment:
```suggestion
    ///         An expression that represents an SQL column.
```

(or "represents a column in SQL")